File: in_Java/0551 Student Attendance Record I.java
class Solution {
    public boolean checkRecord(String s) {
        int a = 0;
        for(int i = 0; i < s.length(); i++){
            if(s.charAt(i) == 'A') a++;
            if(i+3 <= s.length() && s.substring(i,i+3).equals("LLL")) return false;
        }
        if(a > 1) return false;
        
        return true;
    }
}

File: in_Python/0551 Student Attendance Record I.py
class Solution(object):
    def checkRecord(self, s):
        """
        :type s: str
        :rtype: bool
        """
        c1 = 0
        for i in range(len(s)):
            if s[i] == "A":
                c1 += 1
            if i < len(s)-2 and s[i:i+3] == "LLL":
                return False
        return c1 < 2